site/app/views/forum/ForumThreadView.php

```diff
@@ -1074,8 +1085,8 @@ public function filter_post_content($original_post_content) {
      * } $author_info
      * @param string[] $post_attachments
      */
-    public function createPost(string $first_post_author_id, array $thread, array $post, $unviewed_posts, $first, $reply_level, $display_option, int $counter, bool $isLiked, $includeReply, array $author_info, array $post_attachments, bool $has_history, bool $is_merged_thread, bool $render = false, bool $thread_announced = false, bool $isCurrentFavorite = false) {
 
+    public function createPost(string $first_post_author_id, array $thread, array $post, $unviewed_posts, $first, $reply_level, $display_option, int $counter, bool $isLiked, bool $taTrue, $includeReply, array $author_info, array $post_attachments, bool $has_history, bool $is_merged_thread, bool $render = false, bool $thread_announced = false, bool $isCurrentFavorite = false) {
```

Comment:
Make sure this argument gets added in ForumController in the getSinglePost function. I had to make a bugfix PR for initial upducks. See https://github.com/Submitty/Submitty/pull/10388 to see where you need to add to.